dallinger/experiment_server/experiment_server.py

```diff
@@ -319,7 +319,10 @@ def advertisement():
         status = part.status
     except exc.SQLAlchemyError:
         status = None
-
+    try:
+        app_id = config.get('id')
```

Comment:
@jessesnyder @davisagli Jesse's two cases are the distinction I was trying to make — default value if none is set vs. default behavior is none is set. In this PR, it's not that there is a default ID given to experiments without one, it's that there is default behavior if no experiment ID exists.